index.d.ts

```diff
@@ -21,6 +21,7 @@ log(chalk.red.bgBlack(chalkTemplate`2 + 3 = {bold ${2 + 3}}`));
 ```
 */
 export default function chalkTemplate(text: TemplateStringsArray, ...placeholders: unknown[]): string;
+export function chalkStderr(text: TemplateStringsArray, ...placeholders: unknown[]): string;
```

Comment:
This needs a separate doc comment with the stderr example. Same with `templateStderr`.